### ISSUE TITLE
Remove Sidenav Component

### ISSUE THREAD

**Issue by ashleyryan** [vmware-clarity/ng-clarity#316]
## Summary

The Sidenav component has been marked as deprecated for quite some time and has a number of open accessibility issues, particularly around color contrast. It should be removed

https://clarity.design/documentation/sidenav
